.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Pip Upload
+
+on:
+  release:
+    types: [created]
```

Comment:
What was the difference between created and published?

Reply:
Type ‘published’ will be triggered if the release is published, no matter it’s draft or not. Type ‘created’ will be triggered when a NON-draft release is created and published.

If you create a new draft release, the same tag is NOT created, it’s not a tagged release. You need to publish it or create a non-draft release.